docker-compose.yml

```diff
@@ -72,6 +72,21 @@ services:
     depends_on:
       - vault
 
+  product_incentory_db:
+    image: postgres:13-alpine
+    container_name: ecomm-product_inventory-db
+    user: postgres
+    volumes:
+      - ./dispute_management_service/.postgres_data:/var/lib/postgresql/data
+    environment:
```

Comment:
You are trying to override the dispute management's database with the products' one